modules/apps/headless/headless-delivery/headless-delivery-impl/src/main/java/com/liferay/headless/delivery/internal/dto/v1_0/util/PageElementUtil.java

```diff
@@ -92,14 +90,9 @@ private static PageElement _toPageElement(
 	static {
 		Bundle bundle = FrameworkUtil.getBundle(PageElementUtil.class);
 
-		BundleContext bundleContext = bundle.getBundleContext();
-
 		_serviceTrackerMap = ServiceTrackerMapFactory.openSingleValueMap(
-			bundleContext, LayoutStructureItemMapper.class, null,
-			ServiceReferenceMapperFactory.create(
-				bundleContext,
-				(layoutStructureItemMapper, emitter) -> emitter.emit(
-					layoutStructureItemMapper.getClassName())));
+			bundle.getBundleContext(), LayoutStructureItemMapper.class,
+			"layout.structure.item.mapper.type");
```

Comment:
by the way, this code is broken now. In line 78, the get is made by passing the class name.

Reply:
please fix that, the property value should be the classname